tests/e2e/vmlistener_test.go

```diff
@@ -346,6 +352,159 @@ func (s *VMListenerTestParams) TestFailoverAcrossVmOnVsan(c *C) {
 	misc.LogTestEnd(c.TestName())
 }
 
+// TestVolumeAttachedForHostdRestart - verify an in use volume
+// remains attached when hostd is restarted.
+// 1. Restart hostd
+// 2. Verify volume stays as attached
+// 3. Kill the vm
+// 4. Status of the volume should be detached.
+func (s *VMListenerTestParams) TestVolumeAttachedForHostdRestart(c *C) {
+	misc.LogTestStart(c.TestName())
+
+	// Create the volume
+	out, err := dockercli.CreateVolume(s.vm1, s.volumeName)
```

Comment:
> @shuklanirdesh82 was this considered when approving the changes for #1550.

@govint #1550 was merged by the time it was noticed. some comments were supplied see https://github.com/vmware/docker-volume-vsphere/pull/1550#pullrequestreview-48515254 and https://github.com/vmware/docker-volume-vsphere/pull/1550#pullrequestreview-48515565